week3/Borko/memory_profiler.py

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+
+import os
+import cProfile
+from pyprof2calltree import convert
```

Comment:
You should have included that in a requirements file as a dependency

Reply:
Ok I will fix that.